zenodo/modules/deposit/mappings/v7/deposits/records/record-v1.0.0.json

```diff
@@ -222,22 +222,22 @@
         "type": "date"
       },
       "title": {
-        "type": "text"
+        "type": "keyword"
```

Comment:
Based on https://github.com/zenodo/zenodo/pull/2446#issuecomment-1479311413, I would say we keep this as `text`